### DISCUSSION TITLE
Can I reuse the RFC template and readme?

### DISCUSSION THREAD

**Discussion by kushsharma** [cockroachdb/cockroach#71379] [Q&A] [answered]
I really like the way RFCs are being added to cockroachdb repository as it helps me understand why a certain decision was taken and wanted to have the same structure in one of my projects. Is it okay if I reuse the `template` and `readme` with some minimal changes? Or will it make people angry that I copied this from here? 😄

**Comment by jordanlewis** [comment, marked as answer]
Go ahead! You're welcome to use them as templates. You can credit CockroachDB if you like.
